src/main/java/com/bobocode/svydovets/annotation/Component.java

```diff
@@ -8,4 +8,6 @@
 @Target(ElementType.TYPE)
 @Retention(RetentionPolicy.RUNTIME)
 public @interface Component {
+
+    String value() default "";
```

Comment:
value() -> name()

Reply:
Disided to keep value because it only one field in this annotation and spring does the same)